src/test/java/uk/gov/ida/eventemitter/EventEmitterIntegrationTest.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.ida.eventemitter;
+
+import cloud.localstack.LocalstackTestRunner;
+import cloud.localstack.TestUtils;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.CreateQueueRequest;
+import com.amazonaws.services.sqs.model.Message;
+import com.amazonaws.services.sqs.model.ReceiveMessageRequest;
+import com.google.inject.AbstractModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.Provides;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@RunWith(LocalstackTestRunner.class)
+public class EventEmitterIntegrationTest {
+
+    private static final String QUEUE_NAME = "queueName";
+    private static Injector injector;
+    private static String queueUrl;
+    private static AmazonSQS clientSqs;
+
+    @BeforeClass
+    public static void setUp() {
```

Comment:
Once we have a module for production, we should be able to re-use it for tests.

Reply:
Add a simple Event Emitter module.

Reply:
can we use the `EventEmitterModule` to setUp this test?

That way we 
a) save ourselves some time by reusing code
b) make the tests closer to production use (and therefore better)

Reply:
Done.